dev_tools/oneapi/add_host_function.sh

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+
+extract_varname() {
+    local str="$1"
+    # local GET_PARAM=" *([^ ]*) *$"
+    # Need to remove the  = ....
+    # note. it only remove the simple one
```

Comment:
Here you mean that it doesn't work with `x = y = 3` sort of cases?

Reply:
Yes, but I think the template will not have this case?